addons/mail/static/src/components/chatter_topbar/chatter_topbar.scss

```diff
@@ -3,103 +3,41 @@
 // ------------------------------------------------------------------
 
 .o_ChatterTopbar {
-    display: flex;
-    flex-direction: row;
-    justify-content: space-between;
     // We need the +1 to handle the status bar border-bottom.
     // The var is called `$o-statusbar-height`, but is used on button, therefore
     // doesn't include the border-bottom.
     // We use min-height to allow multiples buttons lines on mobile.
     min-height: $o-statusbar-height + 1;
 }
 
-.o_ChatterTopbar_actions {
-    border-bottom: $border-width solid;
-    display: flex;
-    flex: 1;
-    flex-direction: row;
-    flex-wrap: wrap-reverse; // reverse to ensure send buttons are directly above composer
-}
-
 .o_ChatterTopbar_button {
     margin-bottom: -$border-width; /* Needed to allow "overriding" of the bottom border */
 }
 
-.o_ChatterTopbar_buttonAttachmentsCountLoader {
-    margin-left: map-get($spacers, 1);
-}
-
-.o_ChatterTopbar_buttonCount {
-    padding-left: map-get($spacers, 1);
-}
-
 .o_ChatterTopbar_buttonClose {
-    display: flex;
-    flex-shrink: 0;
-    justify-content: center;
-    align-items: center;
     width: 34px;
     height: 34px;
```

Comment:
Not a coincidence that `34px` is exactly  `$o-statusbar-height + 1;` ;)

I suspect that some rules are here for historical reasons, see the `align-items: center;`...  not necessary if we want the button to be aligned with the others.

I propose:
```scss
.o_ChatterTopbar_buttonClose {
    width: $o-statusbar-height;
}
```
```xml
<button class="o_ChatterTopbar_buttonClose btn btn-dark  rounded-lg-bottom flex-shrink-0"/>
```